GameDefinition/Content/ItemKindActor.hs

```diff
@@ -377,7 +376,7 @@ hyena = ItemKind
   , iaspects = [ AddMaxHP 20, AddMaxCalm 70, AddSpeed 32, AddNocto 2 ]
   , ieffects = []
   , ifeature = [Durable, Identified]
-  , idesc    = ""
+  , idesc    = "Skulking in the shadows, waiting for easy prey."
```

Comment:
Perfect pun on "spotting" an animal that hides in shadows. :) The noctovision fits as well.